packages/expo-updates/ios/EXUpdates/EXUpdatesAppController.m

```diff
@@ -75,19 +74,27 @@ - (void)setConfiguration:(NSDictionary *)configuration
   [self resetSelectionPolicyToDefault];
 }
 
+- (EXUpdatesSelectionPolicy *)selectionPolicy
+{
+  if (!_selectionPolicy) {
+    _selectionPolicy = _defaultSelectionPolicy;
+  }
+  return _selectionPolicy;
+}
+
 - (void)setNextSelectionPolicy:(EXUpdatesSelectionPolicy *)nextSelectionPolicy
 {
   _selectionPolicy = nextSelectionPolicy;
 }
 
 - (void)resetSelectionPolicyToDefault
 {
-  _selectionPolicy = [self _defaultSelectionPolicy];
+  _selectionPolicy = _defaultSelectionPolicy;
```

Comment:
I raised a question here: https://github.com/expo/expo/pull/13113#pullrequestreview-673698023 about whether we want this:
```suggestion
  _selectionPolicy = nil;
```